packages/js/plugins/uri-resolvers/ipfs-resolver/polywrap.plugin.yaml

```diff
@@ -0,0 +1,10 @@
+format: 0.0.1-prealpha.3
+name: Ipfs
+language: plugin/typescript
+module: ./src/index.ts
+schema: ./src/schema.graphql
+import_redirects:
+  - uri: "ens/uri-resolver.core.polywrap.eth"
+    schema: ../../../../core-interfaces/uri-resolver/src/query.graphql
```

Comment:
This is referenced in multiple PRs right now - If it's okay with you, I'd do this as part of an end-of-PRs "cleanup PR" which would redo some of the folder structure, and change naming. We should also think about building the interface rather than referencing directly from `src` - something I've done in the FS MR (#912) and this one for their respective interfaces.